core/genesis_write.go

```diff
@@ -143,6 +144,10 @@ func WriteGenesisBlock(tx kv.RwTx, genesis *types.Genesis, overrideCancunTime, o
 		if config.IsOptimism() && overrideOptimismHoloceneTime != nil {
 			config.HoloceneTime = overrideOptimismHoloceneTime
 		}
+		if config.IsOptimism() && overrideOptimismIsthmusTime != nil {
+			config.IsthmusTime = overrideOptimismIsthmusTime
+			config.PragueTime = overrideOptimismIsthmusTime
```

Comment:
Can we add warning message that prague hardfork time is overriden? Like the above code for cancun/ecotone hardfork time overrides